Meadowlark-js/tests/e2e/setup/EnvironmentConfig.ts

```diff
@@ -7,30 +7,39 @@ import { Network, StartedNetwork } from 'testcontainers';
 import * as ApiContainer from './containers/ApiContainer';
 import * as MongoContainer from './containers/MongoContainer';
 import * as OpenSearchContainer from './containers/OpenSearchContainer';
+import * as PostgreSqlContainer from './containers/PostgresqlContainer';
 import { endLog } from './LogConfig';
 
 export async function stop() {
   endLog();
   console.info('-- Tearing down environment --');
   await Promise.all([MongoContainer.stop(), ApiContainer.stop(), OpenSearchContainer.stop()]);
+  if (process.env.DOCUMENT_STORE_PLUGIN === '@edfi/meadowlark-postgresql-backend') {
+    console.info('-- Tearing down postgres --');
+    await PostgreSqlContainer.stop();
+  }
 }
 
-export async function configure() {
+export async function configure(initialize = true) {
   let network: StartedNetwork;
   try {
     network = await new Network().start();
   } catch (error) {
     throw new Error(`\n${error}`);
   }
 
-  if (process.env.DEVELOPER_MODE) {
+  if (!initialize) {
     console.warn(
-      '⚠️ WARNING: Running in DEVELOPER MODE. Containers should be already be started, if not, setup with `test:e2e:dev:setup`⚠️',
+      '⚠️ WARNING: Skipping initialization. Containers should be already be started, if not, setup with `test:e2e:dev:setup`⚠️',
     );
   } else {
-    process.env.ALREADY_SET = 'true';
     console.info('-- Setting up containers --');
     await Promise.all([MongoContainer.setup(network), ApiContainer.setup(network), OpenSearchContainer.setup(network)]);
+
+    if (process.env.DOCUMENT_STORE_PLUGIN === '@edfi/meadowlark-postgresql-backend') {
```

Comment:
I read this line and asked myself, why not a condition before starting MongoDB?

Might be worth adding a comment above line 37 that MongoDB always runs so that we can support authentication.